starkinfra/issuingtokenrequest/__issuingtokenrequest.py

```diff
@@ -0,0 +1,40 @@
+from starkcore.utils.subresource import SubResource
+from starkinfra.utils import rest
+
+
+class IssuingTokenRequest(SubResource):
+    """# IssuingTokenRequest object
+        The IssuingTokenRequest object displays the information of the Token Requests created for your card tokenization.
+        ## Parameters (required):
+        - card_id [string]: card id to be tokenized. ex: "5734340247945216"
+        - wallet_id [string]: desired wallet to be integrated. ex: "google"
+        - method_code [string]: method CODE. ex: "app", "manual"
+        ## Attributes (return-only):
+        - content [string]: token request content. ex: "eyJwdWJsaWNLZXlGaW5nZXJwcmludCI6ICJlNTNiZThjZTRhYWQxNzE5NWU2OWNmMjExOTA5Mjk4YzJkOTE0ODk2IiwgImVuY3J5cHRlZERhdGEiOiAiY2Y3ODNjZDcyNzM2ODA4OWMyYTY4YTQyYTcyMWU5ZmUwYzU5YjMzMzdlZDA0YWRjNTlhZGU5NzY3YmU3OTRjNTUzMDM3MDIzMTRkMTQ1MGNlYzZhZWFhNzdiNjAxZDlkYTlkNDNkYjkzOWNmNTU4ZTNlYWI2OGE1Y2QwODBiNTUzNGFlN2UxZGU1MWYzODBmMWU4MzU2N2M0ODc0MGQ5MjkyNzM5NzVkMTU4MjViNTM2YTc3YjAwMTRjNGQxYjY5MWY2YTIxZDUwYmI5YzYzNjg3Njg0NzM5MWY3NTA3MzJlN2I4MTBlNGRiMzdhNTA0YjM4OGUwY2ZjMGMwODhkMTk0NTEwMTU4OGIxZDA5YTk4OTE3MDNjOGIxZjFhYTQ4YTY4NDA3OTJmZWRkZDcxY2E2NjhlYWY4Mjg5ZDU3MGMiLCAiZW5jcnlwdGVkS2V5IjogIjkzMzVjMDM0ZGIyNmE3NTc3NGNmMGI4MGU2ZjdmMGYwYzcxZjJhZWUzYTFmZjkwYTUwMWExZWNjNGE0YTgwMjE4ZTU0ZGRmMzUxZWY0N2Q4N2Q2ZWZhODRhMjZmMWJkZmE2OGIzZjM4ZjJiOGM4ZWVkYjQ1MTAxODJmMWRiMzllNWNiNWVkZjU4NTExNDEwMGU2ZDRlZWNhM2IwYzk3MzljYmZmMjI5OWRmYjNhNzNmM2E4ZWY5NDRjNTM3ZmQ3OWU0ZmIxNmZlMjlkNTQzNWZmNTIxY2JkMDJiMDRhNTM0YzUzYjg3ZWEwZTdiYTQxNzE1ZTE5NTNkNDUyOWQ3NDExNmZkZTcyMmVlZjU0ZjJhMDMxMDMxMjI2MDIyYWVhNDdiMzY5YTllYmVjY2E1NmY2ZmFhODVkNGVmZWJhYjQwNjVhMDZkMTNhMjMwMzJlODNlODM1YjA5ZjlkZWViNTY0OWE0YzkzMDcyYWYyMDMwNzQyNDQ2MGQxMmYyNjEwNTM0NTYwZGNjMjJmNmM5ZTM2ZDM3YWFiMmVkNjg4NmZjNzQ2OTc1MzY5ODQyODJiMTFmMGZmYmE2NGNjMjQ5ZmEwNDAyYTkzY2RlZjc5MzgzY2VjMDdkMDg3Nzc4MjIzZTNjMjVmYmIyNGRmZjU4ZjNiYWEwN2E5M2I0N2NhMmVhIiwgIm9hZXBIYXNoaW5nQWxnb3JpdGhtIjogIlNIQTI1NiIsICJpdiI6ICJlYjZhZTIwYTQzMWM3OTMyZDNkNGRkOGEyMGUxNGU2NSJ9"
+        - signature [string]: token request signature. ex: "eyJzaWduYXR1cmUiOiAicDJJTFdjQnY4MVRmV3FIMzJNUWFnZWJiMlNMMnZWcEp3bExweE1wMjE2NktKc1V4SGpteFdpVVE4eWp2aDdqUE5HRlNqd0FLVWVDbUJmRzhEbWNiK0h0eDhEbEx0cVpRckVEbjBsOVZtWHMyRXJNdG1JMitvODUvZHRTWlNIbE11ZjBCN0dLM1g1elN0eXVWQ01HaytPL2FoZDFrVERVQjJ0NG0ycm1zb1d2Qi9IajZUQUFBc1pJckdNSjFJRVpNbDhYR0hjNk9aVjc0S2tBN1F5SVZHTHRMSzczbUNqSFI5a1ozL0FJbm1mbEF1dXBNRnpUSlBTSG5OV0F6ems5SDJuZlVPd2JmSktXUTB3LzNsZ01EWTdkZTNPSTA2YUFId082bjI4KzErWDNSZWV0ZlYrQnNtUEUxelkyRVB4OWtiWkIwbDV4bk5XU3dQZUN6VFN3NEtBPT0iLCAiZGF0YVZhbGlkVW50aWxUaW1lc3RhbXAiOiAiMjAyMy0wOC0wM1QyMToyMjoyMFoiLCAidmVyc2lvbiI6ICIzIiwgImluY2x1ZGVkRmllbGRzSW5PcmRlciI6ICJkYXRhVmFsaWRVbnRpbFRpbWVzdGFtcHxhY2NvdW50TnVtYmVyfGFjY291bnRFeHBpcnkiLCAic2lnbmF0dXJlQWxnb3JpdGhtIjogIlJTQS1TSEEyNTYifQ"
+        - metadata [dictionary]: dictionary object used to store additional information about the IssuingTokenRequest object. ex: { authorizationId: 'OjZAqj' }.
```

Comment:
dictionary object used to store additional information about the IssuingTokenRequest object.